### PR TITLE
Vcluster/developerportalupdate

### PR DESCRIPTION
**Pull request by TopSwagCode**

### CODE REVIEW

**Review by sahma19 — APPROVED** [review]

File: domains/certificates/Query.API/API/ContractService/Clients/MeteringPointsClient.cs [hunk 1]
```diff
@@ -118,7 +118,7 @@ public enum MeterType
     Child
 }
 
-public record Technology(string AibFuelCode, string AibTechCode);
+public record Technology(string FuelCode, string TechCode);
```
Comment (martinhenningjensen) [inline domains/certificates/Query.API/API/ContractService/Clients/MeteringPointsClient.cs:121]: Why was this changed?

File: domains/measurements/Measurements.API/API/MeteringPoints/Api/Dto/Responses/Technology.cs [hunk 2]
```diff
@@ -1,3 +1,3 @@
 namespace API.MeteringPoints.Api.Dto.Responses;
 
-public record Technology(string AibTechCode, string AibFuelCode);
+public record Technology(string FuelCode, string TechCode);
```
Comment (martinhenningjensen) [inline domains/measurements/Measurements.API/API/MeteringPoints/Api/Dto/Responses/Technology.cs:3]: Why was this changed?

File: domains/measurements/Measurements.API/API/Program.cs [hunk 3]
```diff
@@ -13,8 +15,15 @@
...
-    await app.RunAsync();
-    Log.Information("Server stopped.");
+    if (args.Contains("--swagger"))
+    {
+        app.BuildSwaggerYamlFile(app.Environment, "measurements.yaml"); // Lets build
+    }
+    else
+    {
+        await app.RunAsync();
+        Log.Information("Server stopped.");
```
Comment (martinhenningjensen) [inline domains/measurements/Measurements.API/API/Program.cs:25]: Is this also logged in other sub systems?

**Review by martinhenningjensen — DISMISSED** [review]

File: domains/authorization/Authorization.API/API/Authorization/Controllers/ClientController.cs [hunk 4]
```diff
@@ -27,18 +29,6 @@ public ClientController(IMediator mediator, IdentityDescriptor identityDescripto
         _identityDescriptor = identityDescriptor;
     }
 
-    [HttpGet]
-    [Route("api/authorization/client/{idpClientId}")]
-    [SwaggerOperation(
-        Summary = "Retrieves Client",
-        Description = "Retrieves info for client with id idpClientId"
-    )]
-    public async Task<ActionResult<ClientResponse>> GetClient([FromServices] ILogger<ClientResponse> logger, [FromRoute] Guid idpClientId)
```
Comment (tnickelsen) [inline domains/authorization/Authorization.API/API/Authorization/Controllers/ClientController.cs:36]: I dunno if this should be deleted

File: domains/libraries/EnergyOrigin.Setup/EnergyOrigin.Setup/ApiVersions.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+public static class ApiVersions
+{
+    public const string Version20240515 = "20240515"; // Used by frontend
+    public const string Version20230101 = "20230101"; // Used by frontend
+    public const string Version20240423 = "20240423";
+    public const string Version20240110 = "20240110";
+    public const string Version20240103 = "20240103";
+
+    public const int Version20230101AsInt = 20230101;
```
Comment (tnickelsen) [inline domains/libraries/EnergyOrigin.Setup/EnergyOrigin.Setup/ApiVersions.cs:11]: Are these used? Maybe say string to int instead?

File: domains/measurements/Measurements.API/API/Properties/launchSettings.json [hunk 6]
```diff
@@ -1,31 +1,31 @@
-{
```
Comment (tnickelsen) [inline domains/measurements/Measurements.API/API/Properties/launchSettings.json:1]: Why changes to this file?

Comment (TopSwagCode) [reply domains/measurements/Measurements.API/API/Properties/launchSettings.json:1]: dotnet format did it. I had a --swagger to test it worked. But somehow me touching it made it change, when I changed it back.

**Review by martinhenningjensen — APPROVED** [review]

File: domains/authorization/Authorization.API/API/Authorization/Controllers/ClientController.cs (hunk 4 above)
Comment (TopSwagCode) [reply domains/authorization/Authorization.API/API/Authorization/Controllers/ClientController.cs:36]: I have :D

**Review by martinhenningjensen — APPROVED** [review]

**Review by martinhenningjensen — APPROVED** [review]
